keras/optimizers/optimizer_test.py

```diff
@@ -7,6 +7,36 @@
 
 
 class OptimizerTest(testing.TestCase):
```

Comment:
Can we also update the test in tensorflow/optimizer_distribute_test.py for distribution related test case? (for ema and gradient accumulation)